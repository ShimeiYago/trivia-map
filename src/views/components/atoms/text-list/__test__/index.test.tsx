import { shallow, ShallowWrapper } from 'enzyme';
import { TextList, Props } from '..';

let wrapper: ShallowWrapper;

const props: Props = {
  list: ['xxx', 'yyy'],
};

describe('Shallow Snapshot Tests', () => {
  beforeEach(() => {
    wrapper = shallow(<TextList {...props} />);
  });

  it('base', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
