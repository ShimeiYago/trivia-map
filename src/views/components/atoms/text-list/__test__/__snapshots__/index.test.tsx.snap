// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Shallow Snapshot Tests base 1`] = `
<ForwardRef(Stack)
  className="ul"
  component="ul"
  spacing={1}
>
  <ForwardRef(Typography)
    component="li"
    key="text-list-0"
  >
    xxx
  </ForwardRef(Typography)>
  <ForwardRef(Typography)
    component="li"
    key="text-list-1"
  >
    yyy
  </ForwardRef(Typography)>
</ForwardRef(Stack)>
`;
